content/en/platform/corda/4.11/community/legal-info/legal-info-4.11.html

```diff
@@ -5,10 +5,13 @@
 - /HEAD/legal-info.html
 - /legal-info.html
 date: '2023-11-24'
+project: corda
+section_menu: corda-community-4-11
+version: 'Community & Open Source 4.11'
 menu:
   corda-community-4-11:
-    identifier: corda-community-4-11-legal-info
-    parent: corda-community-4-11-release-notes
+    identifier: corda-community-4-11-0-legal-info
+    parent: corda-community-4-11-legal-info
     weight: 450
     name: Third-party software licence information - 4.11
```

Comment:
```suggestion
    name: 4.11
```
Think that's what you did in the other versions?